packages/react-native/src/transport.ts

```diff
@@ -33,7 +33,8 @@ export class Transport implements Types.Transport {
   }
 
   private buildUserAgent() {
-    const reactNativeVersion = `${Platform.constants.reactNativeVersion.major}.${Platform.constants.reactNativeVersion.minor}.${Platform.constants.reactNativeVersion.patch}`
+    const rnVersion = Platform.constants?.reactNativeVersion
+    const rnVersionStr = `${rnVersion?.major}.${rnVersion?.minor}.${rnVersion?.patch}`
```

Comment:
How about we at least check for `rnVersion`?
```suggestion
    const rnVersionStr = rnVersion ? `${rnVersion?.major}.${rnVersion?.minor}.${rnVersion?.patch}` : 'n/a'
```